src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -115,12 +134,16 @@
                                                      :display-name @full-name
                                                      :color        @custom-color}])
           :style                     style/continue-button
-          :disabled                  (and (not (seq @full-name)) (not @validation-msg))}
-         (i18n/label :t/continue)]]])))
+          :disabled                  (or (not (seq @full-name)) @validation-msg)}
+         (i18n/label :t/continue)]]])
+    (finally
+     (fn []
```

Comment:
I'm not sure this function is being executed. Could you please make sure this fn is called?

 AFAIK the `fn ` is not needed, so I'd remove it, I mean, do:

```clojure
(finally
  instruction-1
  Instruction-2
  ,,,  )
```

Instead of:

```clojure
(finally
  (fn []
    instruction-1
    Instruction-2
    ,,,  ))
```